packages/core/src/App/Containers/RootComponent/root-component.jsx

```diff
@@ -61,7 +61,17 @@ const RootComponent = observer(props => {
             sessionStorage.removeItem('active_wallet_loginid');
             localStorage.setItem('client.accounts', '{}');
             localStorage.removeItem('active_wallet_loginid');
-            window.location.assign(redirectUrl);
+
+            const redirect_to_lowcode = sessionStorage.getItem('redirect_to');
```

Comment:
Is this consumed in Traders Hub low code? In Deriv.app we use `tradershub_redirect_to` key